SU-SWS/stanford_components
# Earth 291

# READY FOR REVIEW

# Summary
- EARTH-291: Featured Blocks Paragraph Type

# Needed By (Date)
- July release

# Urgency
- Normal

# Steps to Test

1. Check out branch EARTH-291 in stanford_paragraph_types, stanford_components, and SE3_BLT
2. Run `drush cim -y`
3. Run `drush csim local -y`
4. Run `drush features-import-all -y`
5. Navigate to a complex page and create a new `Featured Blocks` paragraph type
6. Review display

# Associated Issues and/or People
- EARTH-291
- Contains the centered container logic that we might use for other sections going forward

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
